proto/svc-storage-grpc-flight_plan.proto

```diff
@@ -73,46 +73,50 @@ message Data {
     string pilot_id = 1;
     // vehicle_id UUID v4
     string vehicle_id = 2;
-    // the path of the flight
-    grpc.geo_types.GeoLineStringZ path = 3;
+    // the waypoints of the flight
+    grpc.geo_types.GeoLineStringZ waypoints = 3;
+    // cruise speed of the vehicle
+    float cruise_speed = 4;
+    // hover speed of the vehicle
+    float hover_speed = 5;
     // weather_conditions
-    optional string weather_conditions = 4;
+    optional string weather_conditions = 6;
     // origin_vertiport_id UUID v4, only listed for get results, not needed for
     // creation (known through pad_id)
-    optional string origin_vertiport_id = 5;
+    string origin_vertiport_id = 7;
```

Comment:
Vertiports are optional as they're not actually being saved to the database since they can be determined by the vertipads (a vertipady is -always- part of a vertiport)
Saving the vertiport_id is duplication of data

Reply:
The vertiport_ids are getting injected here https://github.com/aetheric-oss/svc-storage/blob/d4853fef10b2d00adf841d3b79d521702249b976/server/src/resources/flight_plan/mod.rs#L236

This means, everytime the `Data` object is returned from the database, the id's are there (but indeed, as a `Option` value since we don't have/need this information when adding the info to the database which is using the same `Data` struct in it's create call).
I'm not happy with the current solution though, so I would want to change this in the future being a custom query where the vertiport_id is already JOINed in. In that case we can use a seperate struct for the `Data` returned which won't need the `Option` anymore.